app.js

```diff
@@ -65,12 +61,12 @@ app.get('/', function (req, res) {
 app.get(/^\/(\w+)\/hello/, function (req, res) {
   var lang = req.params[0];
   // Just get this string
-  mybundle.getEntryInfo({ resourceKey: 'hello', languageId: lang}, function (err, data) {
+  mybundle.entry({ resourceKey: 'hello', languageId: lang})
+   .getInfo(function (err, entry) {
```

Comment:
I am not sure I understand. Do you mean it will have the target translation data and the source as well?